.appveyor.yml

```diff
@@ -19,7 +20,8 @@ environment:
   TreatWarningsAsErrors: true
   codecov_token: 9a7c2ba3-0a4b-4479-96e8-3bfd01a982f6
 before_build:
-  msbuild src\StreamJsonRpc.sln /nologo /m /v:quiet /t:restore
+- dotnet --info
```

Comment:
I assume this is here for logging so we can see the environment?

Reply:
Yes. It gives us a record of what runtimes are installed on appveyor for a given build. Right now they have quite a few installed -- everything we need.